code/__DEFINES/typecheck/mobs_generic.dm

```diff
@@ -4,6 +4,7 @@
 #define isRemoteControllingOrAI(M)	((M && M.client && M.client.remote_control) || (istype(M, /mob/living/silicon/ai)))
 
 #define isbrain(A) (istype(A, /mob/living/brain))
+#define ishologram(A) (istype(A, /mob/hologram))
```

Comment:
Can you remove it too ? That was my point above, don't want to encourage code behavior around snowflake typechecking, it always bites us back later on